Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM ubuntu:14.04
 
-ENV GOVERSION 1.5.3
-
 RUN apt-get update -y && apt-get install --no-install-recommends -y -q \
                          curl build-essential ca-certificates git mercurial bzr \
                       && rm -rf /var/lib/apt/lists/*
 
+ENV GOVERSION 1.5.4
```

Comment:
You're right, the layers are just diffs, so it shouldn't make a significant difference